fenix/app/src/main/AndroidManifest.xml

```diff
@@ -49,6 +49,10 @@
         android:usesCleartextTraffic="true"
         tools:ignore="UnusedAttribute">
 
+        <profileable
+            android:shell="true"
+            tools:targetApi="29" />
```

Comment:
Without looking too deeply, I'm curious whether adding this to our normal manifest could have any impact on production performance and if so whether there is there an easy way to create a "copy" app that we could use under test.

Reply:
Good question, looking into the docs, the [profileable tag](﻿﻿https://developer.android.com/guide/topics/manifest/profileable-element) makes the non debuggable builds profileable on 29+, it doesn't look like it has any impact on the performance since it's designed to benchmark release builds for more accurate perf metrics.